libs/designer/src/lib/core/queries/connections.ts

```diff
@@ -12,13 +13,9 @@ export interface ConnectorWithParsedSwagger {
 }
 
 export const getConnectorWithSwagger = async (connectorId: string): Promise<ConnectorWithParsedSwagger> => {
-  const { connector, swagger } = await getReactQueryClient().fetchQuery(['apiWithSwaggers', connectorId.toLowerCase()], async () => {
```

Comment:
Since we removed the queryClient, we will have to make sure whoever calls this need to call through queryClient.. even hosts can call this.

Reply:
This still going through a queryClient, the `getConnector` and `getSwagger` calls do it internally